### PR TITLE
Ex1 compendium+gift

### PR DESCRIPTION
**Pull request by Strate-S**
EX1 enemy (and Twins) compendium entries + Tengu Prince gift dialogue

### CODE REVIEW

File: Gaijinizer patch/CommonEvents_175.txt [hunk 1]
```diff
@@ -91,40 +93,40 @@ Lampas were acclaimed?#GT
...
 > CONTEXT: CommonEvents_175/45/Dialogue/enemy
-Thank you♪
-There is not much individual difference in the
-sensitivity of Lampas
-Maybe I'll read the same response...#GT
+Just kidding♪
+Sensitivity doesn't vary much between individual
+Lampas, so I'll probably have the same reaction
+when I read it...#Strate
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_175.txt:100]: Note

File: Gaijinizer patch/Enemies.txt [hunk 2]
```diff
@@ -2906,14 +2910,12 @@ M.O.W生物兵器　■■-■■■■（ロストナンバー）『双子』
...
-M.O.W. Biological weapons ■ ■ ■ ■ ■ ■ ■ ■ (lost number)
-"Twins".
-It is the cause of Mukuro's birth and the cause of the
-ancient disaster.
-Because they have appeared in the same manner as the
-women's stories and the past
-It is speculated that it has the ability to transcend
-space-time.#MTLed
+M.O.W. Biological Weapons ■■-■■■■ (lost number) "Twins".
+The cause of Mukuro's birth and the culprits behind the
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2914]: Do we want to localize "lost number"?

File: Gaijinizer patch/Enemies.txt [hunk 3]
```diff
@@ -2822,14 +2822,18 @@ fit is typical of the Oni race.#Ron
 人間の精気や肉体を糧にし、一度に複数の産卵を行う
 回帰された魂は二度と開放されることは無い。
 > CONTEXT: Enemies/615/BookDescription/3
-
+The progenitor of the Harpy Spawn, who appears during
+breeding season. She feeds on the energy and flesh of
+humans to lay multiple eggs at the same time. A soul that
+was regressed will never be released again. Due to her
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2829]: Note "released".

File: Gaijinizer patch/Enemies.txt (hunk 2 above)
Comment (Strate-S) [reply Gaijinizer patch/Enemies.txt:2914]: Maybe something more machine-like such as "number not found"?

Comment (Triketos) [reply Gaijinizer patch/Enemies.txt]: Yeah... talked about it in server, and this should be good:
```
M.O.W. Biological Weapons ■■-■■■■[ID NOT FOUND] "Twins".
The cause of Mukuro's birth and the culprits behind
the ancient calamity. Due to the way they speak and
the fact that their appearance is the same as it was
in the past, it is speculated that they possess the
ability to transcend time and space.#Strate
```

File: Gaijinizer patch/Enemies.txt [hunk 4]
```diff
@@ -2928,14 +2930,12 @@ space-time.#MTLed
...
-attack.
-While Harpy carries his soul to the underworld, Modoki
-brings it to hell.#MTLed
+A fierce carnivorous demon that roams the city's skies.
+It attacks wounded and unarmed people from above, seeking
+their flesh and blood. On the other hand, they do not
+attack strong people like the protagonist.
+While regular harpies carry the soul to the afterlife,
+Harpy Spawn bring it to Hell instead.#Strate
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2939]: Altered to better fit line limits:
```
A fierce carnivorous demon that roams the skies.
It attacks wounded and unarmed people from above,
seeking their flesh and blood. On the other hand,
they don't attack strong people like the protagonist.
While regular harpies carry the soul to the
afterlife, Harpy Spawn bring it to Hell instead.#Strate
```

File: Gaijinizer patch/Enemies.txt [hunk 5]
```diff
@@ -2822,14 +2822,18 @@ fit is typical of the Oni race.#Ron
 人間の精気や肉体を糧にし、一度に複数の産卵を行う
 回帰された魂は二度と開放されることは無い。
 > CONTEXT: Enemies/615/BookDescription/3
-
+The progenitor of the Harpy Spawn, who appears during
+breeding season. She feeds on the bodies and souls of
+humans to lay multiple eggs at the same time. A soul that
+has converted will never be released again. Due to her
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2829]: to fit a bit better:
```
The progenitor of the Harpy Spawn, who appears during
breeding season. She feeds on the bodies and souls
of humans to lay multiple eggs at the same time. A 
soul that has converted will never be released again.
```

File: Gaijinizer patch/Enemies.txt [hunk 6]
```diff
@@ -2822,14 +2822,18 @@ fit is typical of the Oni race.#Ron
...
 
 > BEGIN STRING
 その風貌や、胎内回帰の特性から
 クイーンを崇める宗教団体もいるのだとか。
 > CONTEXT: Enemies/615/BookDescription/6
-
+appearance and womb regression ability, there are even
+religious groups who worship the Queen.#StrateUnsureLocalization
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2837]: adding on from previous comment:
```
Due to her appearance and womb regression ability,
there are even religious groups who worship the Queen.
```

**Review by Triketos — APPROVED** [review]
Looks good!

**Pull request merged**